ui/src/components/PathList/PathListItem.tsx

```diff
@@ -0,0 +1,129 @@
+import { Accordion, AccordionSummary, AccordionDetails, Grid } from "@mui/material";
+import { useContext, useMemo, useEffect } from "react";
+import { CollapsedContext } from "../../CollapsedContext";
+import ExpandMoreIcon from '@mui/icons-material/ExpandMore';
+import './PathListItem.sass';
+
+export interface PathListItemProps {
+    change: any
+    showChangeType?: string
+}
+
+export const PathListItem: React.FC<PathListItemProps> = ({ change, showChangeType = "" }) => {
+    const changeVal = change.Value
+    const { accordions, setAccordions } = useContext(CollapsedContext);
+    const filteredChanges = useMemo(() => {
+        const changes = showChangeType ? changeVal?.Paths.filter((path: any) => path.Changelog.type === showChangeType) : changeVal?.Paths
+        setAccordions((prev) => {
+            let newArr = [...prev, changes.map((path: any) => { return { isCollpased: true, id: JSON.stringify(path) } })].flat()
+            return newArr
+        })
+        return changes
+    }, [changeVal?.Paths, setAccordions, showChangeType])
+
+    const onClick = (id: string) => {
+        setAccordions((prev) => {
+            let newArr = [...prev]
+            let accordion = newArr.find(x => x.id === id)
+            if (accordion)
+                accordion.isCollpased = !accordion?.isCollpased
```

Comment:
what do you do with this "accordion" variable?

Reply:
change his isCollpased prop.
so he is rendered wih the correct prop